src/docfx/localization/LocalizationUtility.cs

```diff
@@ -237,6 +182,42 @@ public static bool TryRemoveLocale(string name, out string nameWithoutLocale, ou
             return false;
         }
 
+        internal static bool TryGetSourceRepository(string remote, string branch, out string sourceRemote, out string sourceBranch, out string locale)
```

Comment:
nit: Still make it public to make diff smaller?

```suggestion
        public static bool TryGetSourceRepository(string remote, string branch, out string sourceRemote, out string sourceBranch, out string locale)
```

Reply:
or let me try to make it private